uber/config.py

```diff
@@ -136,22 +136,6 @@ def SUPPORTER_COUNT(self):
     def REMAINING_BADGES(self):
         return max(0, self.MAX_BADGE_SALES - self.BADGES_SOLD)
 
```

Comment:
I completely forgot that we had defined a ``c.SQLALCHEMY_URL`` property to make our code forwards compatible with some future Docker work.  This caused two problems:

1) It raised an exception when I deployed to the staging server.  I didn't get that exception locally because I still had a local ``development.ini`` with a ``sqlalchemy_url`` global setting I had forgotten to move into the ``[secret]`` section.  This fixes that problem by having our custom property access ``_config['secret']['sqlalchemy_url']`` instead of ``_config['sqlalchemy_url']``.

2) Even after that's fixed, the fact that this property is defined on the ``Config`` class means that it would have been rendered to Javascript.  By creating a separate ``SecretConfig`` class, we stop that from happening, and we add code to our ``__getattr__`` so that people can still say ``c.SQLALCHEMY_URL`` and have it do the correct thing.